db/migrations/1_create_task_children_data_view.sql

```diff
@@ -0,0 +1,36 @@
+-- +migrate Up
+DROP VIEW IF EXISTS `task_children_data_view`;
+SET @saved_cs_client          = @@character_set_client;
+SET @saved_cs_results         = @@character_set_results;
+SET @saved_col_connection     = @@collation_connection;
+SET character_set_client      = utf8mb4;
+SET character_set_results     = utf8mb4;
+SET collation_connection      = utf8mb4_general_ci;
+CREATE ALGORITHM=UNDEFINED
+  DEFINER=`algorea`@`%` SQL SECURITY DEFINER
+  VIEW `task_children_data_view` AS
+  SELECT
+    `parent_users_items`.`ID` AS `idUserItem`,
+    SUM(IF(((`task_children`.`ID` IS NOT NULL) AND `task_children`.`bValidated`), 1, 0)) AS `nbChildrenValidated`,
+    SUM(IF(((`task_children`.`ID` IS NOT NULL) AND `task_children`.`bValidated`), 0, 1)) AS `nbChildrenNonValidated`,
+    SUM(IF(((`items_items`.`sCategory` = 'Validation') AND
+      (ISNULL(`task_children`.`ID`) OR (`task_children`.`bValidated` = 0))), 1, 0)) AS `nbChildrenCategory`,
+    MAX(`task_children`.`sValidationDate`) AS `maxValidationDate`,
+    MAX(IF((`items_items`.`sCategory` = 'Validation'), `task_children`.`sValidationDate`, NULL)) AS `maxValidationDateCategories`
```

Comment:
Actually in 
```sql
IF((`items_items`.`sCategory` = 'Validation'), `task_children`.`sValidationDate`, NULL))
```
 I would not put the parenthesis around the expression. Just a personal opinion. (it does apply to the whole PR)

Reply:
fixed